land_use/emulator.rst

```diff
@@ -0,0 +1,152 @@
+
+.. _fig-Land-Use_Pathway_Scenario_Matrix:
+.. figure:: /_static/Land-Use_Pathway_Scenario_Matrix.png
+   :width: 800px
+
+   Land-Use Scenario Matrix.
+
+In their entirety, the combination of these distinct land-use pathways provide MESSAGE with a range of biomass potentials available for energy production at different costs, henceforth referred to as BIO-categories, along with the associated land-use related emissions (CO2, CH4 and N2O). The different carbon prices provide MESSAGE with options for mitigating land-use related GHG emissions, henceforth referred to as GHG-categories. The combination of land-use pathways can therefore be depicted as a trade-off surface, illustrated for SSP2 (add reference) in the figure below (add link). The figure depicts global biomass potentials and respective GHG emissions at different carbon prices cumulated from 2010 to 2100.
+.. _fig-CD_Links_SSP2_v2_baseline_Global_LanduseSurface:
+.. figure:: /_static/CD_Links_SSP2_v2_baseline_Global_LanduseSurface.png
+   :width: 800px
+
+   Land-Use Pathway Trade-Off Surface for SSP2.
+
+From the trade-off surface it possible to deduct that for a MESSAGE scenario without climate policy, land-use pathways of the lower BIO-categories and lowest GHG-categories will be used. The energy system will therefore only use biomass for energy production to the extent that it is economically viable without mitigating emissions. When climate policy scenarios are run in MESSAGE, the land-use pathways will be chosen such that the optimal balance between the land-use related emission and biomass use in the energy system is obtained. In addition to serving as a commodity from which energy can be generated, biomass can also be used to obtain negative emissions via BECCS.
+
+
+Equations and constraints
+-------------------------
+
+The land-use pathways are integrated into MESSAGE using a dedicated set of equations (add link to documentation). At the core, these state that the linear combination of land-use pathways must be equal to 1. Therefore, separately for each region, discrete land-use development pathways can either be used individually or combine linearly to obtain biomass quantities for example, which are not explicitly represented as part of the land-use matrix. This also applies to the mitigation dimension i.e. to the GHG categories.
```

Comment:
This sentence "At the core, these state that the linear combination of land-use pathways must be equal to 1. Therefore, separately" and the following discussion is not clear. Is this the sum of land shares from a total land available?